contents/blog/posthog-vs-matomo.md

```diff
@@ -0,0 +1,435 @@
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td><strong>Import</strong></td>
+        	<td></td>
+        	<td></td>
+    	</tr>
+        <tr>
+        	<td>Google Analytics</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>API</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Redshift</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Google Cloud Storage</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Snowflake</td>
+			<td className="text-center"><span className="text-red text-lg">✖</span></td>
+          <td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Amazon S3</td>
+			<td className="text-center"><span className="text-red text-lg">✖</span></td>
+          <td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Sentry</td>
+			<td className="text-center"><span className="text-red text-lg">✖</span></td>
+          <td className="text-center"><span className="text-green text-lg">✔</span></td>        
+    	</tr>
+		<tr>
+        	<td>Hubspot</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+		<tr>
+        	<td>Salesforce</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+		<tr>
+        	<td>Zapier</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Zendesk</td>
+			<td className="text-center"><span className="text-red text-lg">✖</span></td>
+          <td className="text-center"><span className="text-green text-lg">✔</span></td>       
+    	</tr>
+	</tbody>
+</table>
+</div>
+
+## Strengths of PostHog
+
+![PostHog screenshot](../images/blog/gdpr-compliant-analytics/posthog-gdpr-compliant.png)
+
+### Feature Flag functionality
```

Comment:
```suggestion
### Feature Flags + product analytics = massive win
```

Reply:
Have made some tweaks on Simon's suggestion and think we could play around with this heading a bit. Thoughts?